cmd/build/generate.go

```diff
@@ -45,21 +45,23 @@ func (g Generate) All(ctx context.Context) error {
 }
 
 func (Generate) code() error {
+	const paths = "paths="
+
 	err := shr.New(sh.WithWorkDir("apis")).Run("controller-gen",
 		"crd:crdVersions=v1,generateEmbeddedObjectMeta=true",
-		"paths=./core/...",
-		"output:crd:artifacts:config=../config/crds",
+		paths+dotFilepathJoin("core", "..."),
+		"output:crd:artifacts:config="+filepath.Join("..", "config", "crds"),
 	)
 	if err != nil {
 		return fmt.Errorf("generating kubernetes manifests: %w", err)
 	}
 
-	err = sh.New(sh.WithWorkDir("apis")).Run("controller-gen", "object", "paths=./...")
+	err = sh.New(sh.WithWorkDir("apis")).Run("controller-gen", "object", paths+dotFilepathJoin("..."))
 	if err != nil {
 		return fmt.Errorf("generating deep copy methods: %w", err)
 	}
 
-	err = shr.Run("controller-gen", "object", "paths=./internal/...")
+	err = shr.Run("controller-gen", "object", paths+dotFilepathJoin("internal", "..."))
```

Comment:
Not a file path, only use forward slashes

Reply:
Fixed